jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -1121,21 +1126,22 @@ def _load_regions(self, regions, combination_mode=None, max_num_regions=None,
         previous_mode = self.app.session.edit_subset_mode.mode
 
         with self.app._jdaviz_helper.batch_load():
+            # This method can edit a particular subset or create a new subset
+            # and apply the combination modes depending on this argument
+            if edit_subset:
+                self.subset_selected = edit_subset
```

Comment:
can we revert `edit_subset` in the same way as `edit_subset_mode`?

Reply:
We can, I just wasn't sure if it would make as much sense to go back to Subset 1 after creating a few other subsets.

Reply:
my feeling is that if this is ignoring the plugin values, then it should not affect them either 🤷‍♂️   But curious what others think as well